AK/IntrusiveRedBlackTree.h

```diff
@@ -174,7 +174,14 @@ class IntrusiveRedBlackTree : public BaseRedBlackTree<K> {
 
     static V* node_to_value(TreeNode& node)
     {
```

Comment:
Commit message:

> as previously the source was smaller than the destination, which triggers a static_assert().

I believe it was a `__builtin_bit_cast` error, not a `static_assert()`.